### ISSUE TITLE
Zoom in/out buttons with NVDA activate Zoom meetings app

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17409]
## Description
I was testing the current Zoom behavior of the Facility Locator map on Desktop with NVDA. When I tried to use the zoom in and zoom out buttons, the browsers wanted to open the Zoom meetings app which is not at all what we want. I compared this to the mapbox implementation on [CMS.gov here](https://www.cms.gov/priorities/innovation/where-innovation-happening) and it did not happen. The code looks the same from what I can see (same `title` and aria-label` props) but a coworker of mine suggested maybe looking at the helpers.js file and disabling any events bound to those buttons. I will also say that I think this is somewhat new behavior - when I did my original audit last year this wasn't happening.

## Additional Info
- This is not happening with VoiceOver on Mac
- Zoom also tries to open when the user hits Enter/Space on the map itself so the entire map is causing this disruption
- [NVDA keyboard shortcuts list](https://www.nvaccess.org/files/nvdaTracAttachments/455/keycommands%20with%20laptop%20keyboard%20layout.html)
- [Jaws keyboard shortcuts list
](https://www.freedomscientific.com/training/jaws/hotkeys/)

## Acceptance Criteria

- [ ] Zoom functionality works on the map with NVDA (and other screen readers) without triggering the Zoom meeting app
- [ ] Accessibility review in an ngrok or Tugboat

**Comment by laflannery** [comment]
@xiongjaneg This is definitely not great, folks using NVDA cannot zoom or use the map at all and there's a possibility that they may get themselves trapped in some sort of loop because they aren't understanding why this is happening. I know we are close-ish to redoing this but this is an issue that I would push to raise with Michelle and see if there a way we could look into this sooner. I gave it a defect level of 2 but really I want to give it close to a 1.5, it close to what I would consider launch-blocking

**Comment by xiongjaneg** [comment]
From Max: some keyboard shortcuts may have been added in the code that may conflict with NVDA shortcuts. Would want to make sure if that's the case, we wouldn't want to conflict with other screen reader shortcuts as well.

**Comment by xiongjaneg** [comment]
@xiongjaneg Add default shortcuts to this ticket

**Comment by laflannery** [comment]
I did a bunch of research into this and found the following:

- NVDA doesn't understand that the zoom buttons are being focused on, it still thinks I'm on the last phone number in the facility results list, which is why it's trying to open the zoom app because that actually would make sense.
- We are using the `<map>` element. CMS.gov is not and I just changed it locally, I only tested it once and the map actually got very small (meaning I think this might be not a simple fix) but I'm pretty sure the zoom buttons are working if that element is switched to a basic <div> instead. I think the screen reader just can't reliably get inside the `<map>`.

**Comment by laflannery** [comment]
@jilladams Confirmed on Prod. This can be closed
